components/channel_header_dropdown/menu_items/toggle_mute_channel/toggle_mute_channel.test.js

```diff
@@ -67,16 +67,4 @@ describe('components/ChannelHeaderDropdown/MenuItemToggleMuteChannel', () => {
             {mark_unread: NotificationLevels.MENTION}
         );
     });
-
-    it('should be hidden if the channel type is DM', () => {
-        const props = {
-            ...baseProps,
-            channel: {
-                ...baseProps.channel,
-                type: Constants.DM_CHANNEL,
-            },
-        };
-        const wrapper = shallow(<MenuItemToggleMuteChannel {...props}/>);
-        expect(wrapper).toMatchSnapshot();
-    });
```

Comment:
Instead of deleting, I suggest to modify the test to consider the new behavior, something like:
```javascript
    it('should show Mute Channel to all channel types', () => {
        [
            Constants.DM_CHANNEL,
            Constants.GM_CHANNEL,
            Constants.OPEN_CHANNEL,
            Constants.PRIVATE_CHANNEL,
            Constants.ARCHIVED_CHANNEL,
        ].forEach((channelType) => {
            const channel = {
                id: 'channel_id',
                type: channelType,
            };

            const wrapper = shallow(
                <MenuItemToggleMuteChannel
                    {...baseProps}
                    channel={channel}
                />
            );
            expect(wrapper.find(MenuItemAction).props().show).toEqual(true);
            expect(wrapper.find(MenuItemAction).props().text).toEqual('Mute Channel');
        });
    });
```